human_cancer_analysis/2_identify_generic_genes_pathways.py

```diff
@@ -496,6 +497,35 @@ def shift_template_experiment_with_metadatafile(
 )
 # -
 
+# Hypergeometric test:
+# Given N number of genes with K common genes in Crow et al.
+# SOPHIE identifies n genes as being common
+# What is the probability that k of the genes identified by SOPHIE
+# are also common in Crow et al.? What is the probability of drawing
+# k or more concordant genes?
+num_Crow_genes = shared_ranking.shape[0]
+num_generic_Crow_genes = shared_ranking.query(f"{ref_rank_col}>=80.0").shape[0]
+num_generic_SOPHIE_genes = shared_ranking[
+    shared_ranking["Percentile (simulated)"] >= 80.0
```

Comment:
You might consider defining this 80% threshold as a constant at the top of this file, so it's easy to change if you ever wanted to.